model/payload.go

```diff
@@ -15,20 +15,13 @@
 // specific language governing permissions and limitations
 // under the License.
 
```

Comment:
Some of the files get moved in a weird way, e.g. this one moves from `processor/error/processor_test.go` to `model/payload.go`. Could you maybe solve this by applying explicit `git rm` and `git mv`?

Reply:
I wasn't able to fix this easily unfortunately. It's possible to adjust the similarity index, but we also have some renames that are correctly detected as renames and changing the similarity index could jeopardize those. If you think it's important i can another stab at it